python/lsst/meas/algorithms/treecorrUtils.py

```diff
@@ -0,0 +1,205 @@
+from lsst.pex.config import ChoiceField, Config, Field, FieldValidationError
+
+
+class TreecorrConfig(Config):
+    """A Config class that holds some of the parameters supported by treecorr.
+
+    The fields in this class correspond to the parameters that can be passed to
+    any calls to `treecorr` methods, including catalog creation and two-point
+    correlation function calculations. The default values set for the fields
+    are identical to the default values set in v4.3 of `treecorr`.
+
+    A separate config class is used instead
+    of constructing a `~lsst.pex.config.DictField` so that mixed types can be
+    supported and the config can be validated at the beginning of the
+    execution.
+
+    Notes
+    -----
+    This is intended to be used in CalcRhoStatistics class. It only supports
```

Comment:
It's time to generalize this and replace 'CalcRhoStatistics class' with correlations of PSF residuals or something similar.